lib/src/connection.dart

```diff
@@ -121,11 +121,20 @@ class Connection {
     }
   }
 
-  void sendEvent(String eventName, Map<String, String> data) {
+  void sendEvent(
+    String eventName,
+    dynamic data, {
+    String channelName = '',
+  }) {
     final event = {
       'event': eventName,
```

Comment:
```suggestion
      'event': 'client-$eventName',
```

Needs to have the prefix: 

![image](https://github.com/indaband/pusher_channels/assets/1339236/715b56ca-7edc-4efe-b724-972249a76984)